client/app/Streamfinder.jsx

```diff
@@ -68,14 +92,6 @@ class Streamfinder extends React.Component {
 
   updateCache(id, data) {
     this.cache.set(id, data);
-    this.state = {
-
-    };
-  }
-
-  handleSearchIdSwitch(id) {
-    //receives an id after search finishes retrieving new data
-    //sets currentId resets currentID state with received Id
   }
```

Comment:
It's possible I'm looking at this wrong, but I see two opening brackets (line 89 and line 93) but only one closing bracket (line 95). As of now it looks like updateCache is within checkCache. What are the differences here?

Reply:
This is Rob's thing. I'm not sure what he had planned there.